mlabs/src/Mlabs/NFT/Api.hs

```diff
@@ -51,18 +64,18 @@ mkSchemaDefinitions ''NFTAppSchema
 -- ENDPOINTS --
 
 type ApiUserContract a = UserContract NFTAppSchema a
-
 type ApiAdminContract a = AdminContract NFTAppSchema a
 
+--type ApiQueryContract a = Contract (Last QueryResponse) NFTAppSchema Text a
+
 -- | User Endpoints .
 endpoints :: NftAppSymbol -> ApiUserContract ()
```

Comment:
You will need to merge with the new `staging` structure.

Reply:
@cstml merged